apps/server/src/shared/repo/dashboard/dashboard.model.mapper.ts

```diff
@@ -82,7 +82,7 @@ export class DashboardModelMapper {
 
 	async mapDashboardToModel(entity: DashboardEntity): Promise<DashboardModelEntity> {
 		const existing = await this.em.findOne(DashboardModelEntity, entity.getId());
-		const modelEntity = existing || new DashboardModelEntity({ id: entity.getId(), user: entity.getUserId() });
+		const modelEntity = existing || new DashboardModelEntity({ id: entity.getId(), user: entity.getUserId() }); // TODO transform user to reference
```

Comment:
I do not understand this comment, please explain to me why there should be another reference

Reply:
as written now, the user field is filled with a plain userId. This is different from an existing modelentity loaded from database, which will have an actual entity here, or at least a wrapped referencetype that can be used to lazyload the user.

fixing that will require changes to the modelEntity itself however, to properly type what is going on in typescript.

Reply:
ok, understood